patches/node/build_enable_perfetto.patch

```diff
@@ -12,6 +12,19 @@ adding associated guards there should be relatively small.
 
 We should upstream this as it will eventually impact Node.js as well.
 
+diff --git a/filenames.json b/filenames.json
+index 5af3886d8d3d74d31249a4d79030a8373b8dad52..8ab04d0b1b58454c6ea21f33870f9557f3a57b56 100644
+--- a/filenames.json
++++ b/filenames.json
+@@ -739,8 +739,6 @@
+     "src/tcp_wrap.h",
+     "src/timers.h",
+     "src/tracing/agent.h",
+-    "src/tracing/node_trace_buffer.h",
```

Comment:
`src/tracing/node_trace_buffer.h` and `src/tracing/node_trace_writer.h` don't work with Perfetto which is why they are removed

Reply:
Does "don't work" mean it doesn't compile? I assume we lose certain node internal events being added to perfetto traces by removing this.

Reply:
`v8::platform::tracing::TraceBuffer`, which `NodeTraceBuffer` inherits from, is not a part of Perfetto at all: https://github.com/nodejs/node/blob/5ab3140dfbf0dfa33a66a6af62942b4290f370c1/deps/v8/BUILD.gn#L6838-L6848

(same applies to `NodeTraceWriter`)